python/ray/experimental/dag/actor_node.py

```diff
@@ -0,0 +1,117 @@
+import ray
+from ray.experimental.dag.dag_node import DAGNode
+
+from typing import Any, Dict, List, Optional, Tuple
+
+
+class ActorNode(DAGNode):
```

Comment:
As a general point, I feel it makes more sense for this to be a `ClassNode` because `Actor` is an implementation detail of the default way to execute the graph.

Reply:
Other libraries, like Serve, will not use this to create an actor (rather a deployment in this case)

Reply:
Same for TaskNode, should be FunctionNode IMO